Makefile

```diff
@@ -55,7 +55,18 @@ clean:
 man: cowsay.1
 
 cowsay.1: cowsay.1.adoc
-	$(A2X) --format manpage ./cowsay.1.adoc
+	set -ex; \
+	mantmpdir="mantmp-$$$$"; \
+	mkdir -p "$$mantmpdir"; \
```

Comment:
Can we do a `MKDIR_P` Makefile variable like GNU seems to do? Here's what [GNU coreutils]() 9.1 does, at least after I run `./configure` on my macOS 10.14 box:

```
MKDIR_P = /opt/local/bin/gmkdir -p
[...]
  $(MKDIR_P) "$(DESTDIR)$(bindir)" || exit 1; \
```

I figure a regular `mkdir -p` would be a good enough default, instead of detecting a GNU-specific mkdir.

The [GNU Makefile Conventions](https://www.gnu.org/prep/standards/html_node/Utilities-in-Makefiles.html#Utilities-in-Makefiles) say "don't use `mkdir -p`, because it's not supported everywhere", but I think we can disregard that since we're not trying to be super-portable to old or exotic systems.

Reply:
> Can we do a `MKDIR_P` Makefile variable like GNU seems to do?

Good idea. Sorry for missing that.

> I figure a regular `mkdir -p` would be a good enough default, instead of detecting a GNU-specific mkdir.
> 
> The [GNU Makefile Conventions](https://www.gnu.org/prep/standards/html_node/Utilities-in-Makefiles.html#Utilities-in-Makefiles) say "don't use `mkdir -p`, because it's not supported everywhere", but I think we can disregard that since we're not trying to be super-portable to old or exotic systems.

And for those old and/or exotic systems, the user overridable `MKDIR_P` will rise and shine.

Reply:
> And for those old and/or exotic systems, the user overridable MKDIR_P will rise and shine.

Bingo, I guess that's what it's there for. :)

Reply:
Unless... we just run `mkdir "$$mantmpdir"` without the `-p`. That should be quite portable, should it not?

OTOH, we could switch to `mktemp`. :-)